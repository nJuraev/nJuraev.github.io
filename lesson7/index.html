<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<link rel="stylesheet" type="text/css" href="css/reset.css">
	<link rel="stylesheet" type="text/css" href="css/style.css">
	<title>Мой првый сайт</title>
</head>
<body>
	<section class="header" >
		<h1><span>Получите видео инструкцию</span><strong>Как заработать миллион</strong></h1>
	</section>
	<section>
		
		<div class="main">
			<div class="video">
				<iframe width="560" height="309" src="https://www.youtube.com/embed/xXMDUhiIzBE?controls=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
			</div>
			<div class="form">
				<p>
					Введите свой email
				</p>
				<form action="#">
					<input type="text" placeholder="Ваше имя">
					<input type="text" placeholder="Ваш email">
					<button class="my-btn">
						Получить инструкцию
					</button>
				</form>
			</div>	
		</div>
		
	</section>
	<section >
		<!-- Блок с приимуществами-->
		<h2 class="feature-header">Почему это надо сделать сейчас</h2>
		<div class="feature">
			<div class="col">
				<img src="img/01.png" alt="Первое приимущество">
				<p>Первое приимущество</p>
			</div>
			<div class="col">
				<img src="img/02.png" alt="Второе приимущество">
				<p>Второе приимущество</p>
			</div>
			<div class="col">
				<img src="img/03.png" alt="Третье приимущество">
				<p>Третье приимущество</p>
			</div>
			<div class="col">
				<img src="img/04.png" alt="Четвертое приимущество">
				<p>Четвертое приимущество</p>
			</div>
			<button class="my-btn">Получить инструкцию</button>
		</div>
	</section>
</body>
</html>