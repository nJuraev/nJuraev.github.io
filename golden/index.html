
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="/favicon.ico">
    <title>Golden</title>
    <!-- Bootstrap core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">
    <link href="slick/slick.css" rel="stylesheet">
    <link href="slick/slick-theme.css" rel="stylesheet">
    
    
    <!-- Custom styles for this template -->
    <link href="css/style.css" rel="stylesheet">

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
  </head>

  <body>
  	<main class="main">
  		<div class="container">
			<nav class="navbar navbar-expand-md navbar-light"> 
        <a class="navbar-brand golden-brand" href="#">
            Golden
        </a>
        <button class="navbar-toggler " type="button" data-toggle="collapse" data-target="#myNavbar">
            <span class="navbar-toggler-icon"></span>
        </button>
		        
		   
		    <div class="collapse navbar-collapse " id="myNavbar">
		        <ul class="navbar-nav ml-auto mt-2 mt-lg-0">
		            <li class="nav-item active">
		                <a href="#" class="nav-link m-2 menu-item">Home</a>
		            </li>
		            <li class="nav-item">
		                <a href="#" class="nav-link m-2 menu-item">Services</a>
		            </li>
		            <li class="nav-item">
		                <a href="#" class="nav-link m-2 menu-item">Portfolio</a>
		            </li>
		            <li class="nav-item">
		                <a href="#" class="nav-link m-2 menu-item">About</a>
		            </li>
		            <li class="nav-item">
		                <a href="#" class="nav-link m-2 menu-item">Contact</a>
		            </li>
		        </ul>
		    </div>
			</nav>
     
	     	<h1 class="main-header">
	     		<small>Welcome To Our Studio!</small>
				Itâ€™s nice to Meet you
	     	</h1>
	     	<button class="btn tellmemore">
	     		tell me more
	     	</button>
	    </div><!-- /.container -->
  	</main>
    <!-- Services -->
  	<section class="services">
  		<div class="container">
  			<h2 class='title'>Services<small >Proin iaculis purus consequat sem cure.</small></h2>
  			<div class="slider">
  				<div class="service-item ">
  					<img src="img/basket.png" alt="E-commerce">
  					<h3>E-commerce</h3>
  					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
  				</div>
  				<div class="service-item ">
  					<img src="img/laptop.png" alt="Responsive Web">
  					<h3>Responsive Web</h3>
  					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
  				</div>
  				<div class="service-item ">
  					<img src="img/locked.png" alt="Web Security">
  					<h3>Web Security</h3>
  					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
  				</div>
  			</div>
  		</div>
  	</section>
    
    <section class="portfolio">
      <div class="container">
          <h2 class="title">
          Our portfolio<small>Proin iaculis purus consequat sem cure.</small>
        </h2>
        
        <div class="row">
          <div class="col-lg-4 offset-lg-0 col-md-6 offset-md-0 col-sm-10 offset-sm-1 col-xs-10 offset-xs-1 ">
            <div class=portfolio-item>
              <img src="img/p1.png" alt="">
              <h4>Ebony & Ivory</h4>
              <p>Ebony & Ivory</p>
          </div>
          </div>
          <div class="col-lg-4 offset-lg-0 col-md-6 offset-md-0 col-sm-10 offset-sm-1 col-xs-10 offset-xs-1 ">
            <div class=portfolio-item>
              <img src="img/p2.png" alt="">
              <h4>Ebony & Ivory</h4>
              <p>Print Design</p>
            </div>  
          </div>
          <div class="col-lg-4 offset-lg-0 col-md-6 offset-md-0 col-sm-10  offset-sm-1 col-xs-10 offset-xs-1 ">
            <div class=portfolio-item>
              <img src="img/p3.png" alt="">
              <h4>Clever Poster</h4>
              <p>Print Design</p>
            </div>
          </div>
          <div class="col-lg-4 offset-lg-0 col-md-6 offset-md-0 col-sm-10  offset-sm-1 col-xs-10 offset-xs-1">
            <div class=portfolio-item>
              <img src="img/p4.png" alt="">
              <h4>Vinyl Record</h4>
              <p>Product Mock-Up</p>
            </div>
          </div>
          <div class="col-lg-4 offset-lg-0 col-md-6 offset-md-0 col-sm-10 offset-sm-1 col-xs-10 offset-xs-1 ">
            <div class=portfolio-item>
              <img src="img/p5.png" alt="">
              <h4>Treehouse Template</h4>
              <p>Web Design</p>
            </div>
          </div>
          <div class="col-lg-4 offset-lg-0 col-md-6 offset-md-0 col-sm-10 offset-sm-1 col-xs-10 offset-xs-1">
            <div class=portfolio-item>
            <img src="img/p6.png" alt="">
            <h4>Burned Logo</h4>
            <p>Branding</p>
          </div>
        </div>
      </div>
    </section><!--Section portfolio-->
    
    <section class='about'>
      <div class="container">
        <h2 class="title">About us
          <small>Proin iaculis purus consequat sem cure.</small></h2>
      
      <div class="row">
        <div class="about-text-block">
          <h4 class="about-header text-right">July 2010<br>
            Our Humble Beginnings</h4>
          <p class="about-text text-right">Proin iaculis purus consequat sem cure 
            digni ssim. Donec porttitora entum suscipit 
            aenean rhoncus posuere odio in tincidunt. Proin 
            iaculis purus consequat sem cure digni 
            ssim. Donec porttitora entum suscipit.</p>
        </div>
        <div class="about-image-block"><img class="about-image" src="img/ab1.png" alt=""></div>
        <div class="about-empty-block"></div>
      </div>
      <div class="row">
        <div class="about-empty-block"></div>
        <div class="about-image-block"><img class="about-image" src="img/ab2.png" alt=""></div>
        <div class="about-text-block">
          <h4 class="about-header text-left">January 2011<br>
            Facing Startup Battles</h4>
          <p class="about-text text-left">Proin iaculis purus consequat sem cure 
              digni ssim. Donec porttitora entum suscipit 
              aenean rhoncus posuere odio in tincidunt. Proin 
              iaculis purus consequat sem cure digni 
              ssim. Donec porttitora entum suscipit aenean 
              rhoncus posuere odio in tincidunt.</p>
        </div>
      </div>
      <div class="row about-elements">
        <div class="about-text-block">
          <h4 class="about-header text-right">December 2012<br>
            Enter The Dark Days</h4>
          <p class="about-text text-right">Proin iaculis purus consequat sem cure 
            digni ssim. Donec porttitora entum suscipit 
            aenean rhoncus posuere odio in tincidunt. Proin 
            iaculis purus consequat sem cure digni.</p>
        </div>
        <div class="about-image-block"><img class="about-image" src="img/ab3.png" alt=""></div>
        <div class="about-empty-block"></div>
      </div>
      <div class="row">
        <div class="about-empty-block"></div>
        <div class="about-image-block"><img class="about-image" src="img/ab4.png" alt=""></div>
        <div class="about-text-block">
          <h4 class="about-header text-left">february 2014<br>
            Our Triumph</h4>
          <p class="about-text text-left">Proin iaculis purus consequat sem cure 
            digni ssim. Donec porttitora entum suscipit 
            aenean rhoncus posuere odio in tincidunt. Proin 
            iaculis purus consequat sem cure digni 
            ssim. Donec porttitora entum suscipit aenean.</p>
        </div>
      </div>
      <div class="row">
        <div class="about-empty-block"></div>
        <div class="about-image-block"><span class="circle">
          <span class="circle-border">
            Our<br>
            Story<br>
            Continues<br>
            ...
          </span>
        </span></div>
        <div class="about-text-block"></div>
      </div>
      </div>
    </section><!-- Section About-->
    
    <!-- section Our team -->
    <section class="team">
      <div class="container">
        <h2 class="title">
          our amazing team
          <small>Proin iaculis purus consequat sem cure.</small>
        </h2> 
          <div class="row">
            <div class="col-md-4 offset-md-0 col-10 offset-1  text-center">
            <div class="team-member">
                <img src="img/team/t1.jpg" class="rounded-circle team-member-image" alt="">
              <h4 class="name">Kimberly Thompson</h4>
              <p class="spec">Marketer</p>
              <div class="sots">
                <a href="" class="sots-icon"><img src="img/team/twitter-icon.png"></a>
                <a href="" class="sots-icon"><img src="img/team/facebook-icon.png"></a>
                <a href="" class="sots-icon"><img src="img/team/pinterest-icon.png"></a>
              </div>
            </div>
            </div>
            <div class="col-md-4 offset-md-0 col-10 offset-1 text-center">
              <div class="team-member">
                <img src="img/team/t2.jpg" class="rounded-circle team-member-image" alt="">
                <h4 class="name">Rico Massimo</h4>
                <p class="spec">Coder</p>
                <div class="sots">
                  <a href="" class="sots-icon"><img src="img/team/twitter-icon.png"></a>
                  <a href="" class="sots-icon"><img src="img/team/facebook-icon.png"></a>
                  <a href="" class="sots-icon"><img src="img/team/google-icon.png"></a>
              </div>
              </div>
            </div>
            <div class="col-md-4 offset-md-0 col-10 offset-1 text-center">
             <div class="team-member">
                <img src="img/team/t3.jpg" class="rounded-circle team-member-image" alt="">
               <h4 class="name">Uku Mason</h4>
               <p class="spec">Graphic Designer</p>
               <div class="sots">
                 <a href="" class="sots-icon"><img src="img/team/facebook-icon.png"></a>
                 <a href="" class="sots-icon"><img src="img/team/pinterest-icon.png"></a>
               
               </div>
             </div>
            </div>
          </div>
          <div class="row ">
            <div class="col-8 offset-2 text-center">
              <p class=" team-devis">  
                Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit  
              aenean rhoncus posuere odio in tincidunt proin iaculis.
              </p>
            </div>
          </div>
        
        <div class="row mt-3">
          <div class="col-md-3 col-sm-6 col-12 mt-md-2 mt-4">
            <img src="img/team/s1.png" alt="envato">
          </div>
          <div class="col-md-3 col-sm-6 col-12 mt-md-2 mt-4">
            <img src="img/team/s2.png" alt="WordPress">
          </div>
          <div class="col-md-3 col-sm-6 col-12 mt-md-2 mt-4">
            <img src="img/team/s3.png" alt="ae tuts+">
          </div>
          <div class="col-md-3 col-sm-6 col-12 mt-md-2 mt-4">
            <img src="img/team/s4.png" alt="microlancer">
          </div>
        </div>
      </div>
    </section>

    <!--secion contacts -->
    <section class="contacts">
      <div class="container">
        <h2 class="title contacts-title">Contacts us <small>Proin iaculis purus consequat sem cure. </small></h2>
        
          <form>
            <div class="form-row">
            <div class="col-lg-6 offset-lg-0 col-sm-10 offset-sm-1 p-3">
              <div class="form-group"><input placeholder="your name" type="text" class="form-control"></div>
              <div class="form-group"><input placeholder="your email" type="text" class="form-control"></div>
              <div class="form-group"><input placeholder="subject" type="text" class="form-control"></div>
            </div>
            <div class="col-lg-6 offset-lg-0 col-sm-10 offset-sm-1 p-3">
              <div class="form-group"><textarea placeholder="messsage" name="message" id="" cols="30" rows="9" class="form-control"></textarea></div>
            </div>
            <button class="btn tellmemore">Send message</button>
            </div>
          </form>
        
      </div>
    </section>
    <!--secion contacts -->

    <!-- Copyright-->
    <section class="copyright">
      <div class="container">
        <div class="row">
          <div class="col-md-4 col-12"><p>
            Â© Copyright 2014 FreebiesXpress.com
          </p></div>
          <div class="col-md-4 col-12 text-center">
            <div class="sots">
                <a href="" class="sots-icon"><img src="img/team/twitter-icon.png"></a>
                <a href="" class="sots-icon"><img src="img/team/facebook-icon.png"></a>
                <a href="" class="sots-icon"><img src="img/team/pinterest-icon.png"></a>
                <a href="" class="sots-icon"><img src="img/team/google-icon.png"></a>
              </div>
          </div>
          <div class="col-md-4 col-12"></div>
        </div>
      </div>
    </section>  
    <!-- Copyright-->
    <!-- ÐœÐ¾Ð´Ð°Ð»ÑŒÐ½Ð¾Ðµ Ð¾ÐºÐ½Ð¾-->
    <div class="popup">
      
        <div class="popup-content col-3">
          <button class="popup-close">&times;</button>
          <h4 class="popup-header">
            Get instruction
          </h4>
          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deleniti quibusdam laborum, quas eligendi .</p>
          <div class="myform popup-form">
            <form action="#" class="">
              <label for="name">Your name</label>
              <input type="text" class="myform-input" name='name' placeholder="Enter Your name">
              <label for="email">Your name</label>
              <input type="text"  class="myform-input" name='email' placeholder="Enter Your email">
              <label for="name">Description</label>
              <textarea type="text"  class="myform-input" name='description' cols="5" placeholder="Enter description"></textarea>
              <button class="btn btn-warning send-form">Send</button>
          </form>
          </div>
          
        </div>
     
    </div>


    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="js/jquery-3.1.1.min.js"></script>
    <script src="js/bootstrap.min.js"></script>
    <script src="slick/slick.js"></script>
    <script>
      $(document).ready(function(){
        $('.tellmemore').on('click',function(e){
          e.preventDefault();
          $('.popup').fadeIn();

        });
        $('.popup-close').on('click',function(e){
          $('.popup').fadeOut();
        });


        $('.slider').slick({
          slidesToShow:3,
          slidesToScroll:1,
          responsive: [
          
          
          {
            breakpoint: 991,
            settings: {
              slidesToShow: 2,
              slidesToScroll: 1,
              prevArrow: "<button class='prev arrow'></button>",
              nextArrow: "<button class='next arrow'></button>",
            }
          },
          {
            breakpoint: 767,
            settings: {
              slidesToShow: 1,
              slidesToScroll: 1,
               prevArrow: "<button class='prev arrow'></button>",
              nextArrow: "<button class='next arrow'></button>",
            }
          }
          // You can unslick at a given breakpoint now by adding:
    // settings: "unslick"
    // instead of a settings object
  ]
        });
      });
    </script>
  </body>
</html>
